main/webapp/modules/core/langs/translation-es.json

```diff
@@ -566,59 +566,59 @@
-		"one-col": "One column",
-		"prepend-name": "prepend the original column's name to each cell",
-		"follow-by": "followed by",
-		"before-val": "before the cell's value",
-		"ignore-blank": "Ignore blank cells",
-		"fill-other": "Fill down in other columns",
-		"spec-new-name": "Please specify the new key column's name.",
-		"spec-new-val": "Please specify the new value column's name.",
-		"spec-col-name": "Please specify the new column's name.",
-		"spec-separator": "Please specify the separator between original column names and cell values.",
-		"how-many-rows": "How many rows to transpose?",
-		"expect-two": "Expected an integer at least 2.",
-		"columnize": "Columnize by Key/Value Columns",
-		"note-col": "Note Column (optional)",
-		"sel-col-val": "Please select one key column and one value column that are different from one another.",
-		"cannot-same": "If specified, the note column cannot be the same as the key column or the value column.",
-		"transp-cell-row": "Transpose cells across columns into rows",
-		"transp-cell-col": "Transpose cells in rows into columns",
-		"columnize-col": "Columnize by key/value columns",
-		"data-type": "Data type:",
-		"number": "number",
-		"boolean": "boolean",
-		"date": "date",
+		"transp-cell": "Transponer Celdas de Columnas a Filas",
+		"from-col": "Desde la Columna",
+		"to-col": "Hacia la Columna",
+		"transp-into": "Transponer dentro",
+		"two-new-col": "Dos nuevas columnas",
+		"key-col": "Columna Clave",
+		"contain-names": "(conteniendo los nombres originales de las columnas)",
+		"val-col": "Columna Valor",
+		"contain-val": "(conteniendo los valores originales de las celdas)",
+		"one-col": "Una Columna",
+		"prepend-name": "Anteponer el nombre original de columna en cada celda",
+		"follow-by": "seguido de",
+		"before-val": "antes del valor de la celda",
+		"ignore-blank": "Ignorar celdas vacias",
+		"fill-other": "Llenar en otras columnas",
+		"spec-new-name": "Por favor indique el nombre la nueva columna clave.",
+		"spec-new-val": "Por favor indique el nombre de columna para el nuevo valor.",
+		"spec-col-name": "Por favor indique el nombre de la nueva columna.",
+		"spec-separator": "Por favor indique el separador entre los nombres de columna originales y los valores de celda.",
+		"how-many-rows": "¿Cuantas filas desea transponer?",
+		"expect-two": "Se espera un numero entero, como minimo 2.",
+		"columnize": "Crear Columnas a partir de Columnas Clave/Valor",
+		"note-col": "Columna para Notas (opcional)",
+		"sel-col-val": "Por favor, seleccione una columna de clave y una columna de valor que sean diferentes la una de la otra.",
+		"cannot-same": "Si se indica, la columna para notas no puede ser la misma que la columna clave o la columna de valor.",
+		"transp-cell-row": "Transponer celdas de columnas a filas",
+		"transp-cell-col": "Transponer celdas de filas a columnas",
+		"columnize-col": "Crear Columnas a partir de Columnas Clave/Valor",
```

Comment:
I think "Crear Columnas a partir de Columnas Clave/Valor" is more intuitive than the new suggestion.  But that's my personal impression.